fabric-rendering-v1/src/main/java/net/fabricmc/fabric/api/client/render/EntityRendererRegistry.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.client.render;
```

Comment:
where is the v1 package?

Reply:
1.14.4 doesn't have rendering v1
just backported rendering v0, and added the hud callback

Reply:
two choices basically
1. turn this into v0
2. fix package to be v1

Reply:
ok wait

Reply:
Fixed